Dockerfile

```diff
@@ -22,6 +22,7 @@ RUN apt-get -qq update \
     unzip \
     openjdk-11-jdk \
     maven \
+    dart \
```

Comment:
Is `dart` the right name here?https://github.com/getsentry/craft/pull/353#issuecomment-1043208865 suggests otherwise ...

Reply:
[libdart6-all-dev](https://dartsim.github.io/install_dart_on_ubuntu.html#install-dart)?

[libdart-all-dev](https://packages.ubuntu.com/search?suite=default&section=all&arch=any&keywords=libdart-all-dev&searchon=names)